src/openforms/authentication/api/views.py

```diff
@@ -46,6 +53,7 @@ class AuthenticationLogoutView(APIView):
         description=_(
             "Calling this endpoint will clear the current user session and delete the session cookie."
         ),
+        deprecated=True,  # replaced with the submission specific SubmissionLogoutView
```

Comment:
Can you please also amend the description of the endpoint with a pointer to the new endpoint. Something like "this endpoint is deprecated, instead use the submission-specific endpoint."